src/popup/module/EmojiSelect.js

```diff
@@ -125,6 +123,8 @@ export async function triggerOnSelect(emoji) {
         if (closePopup) {
             window.close();
         }
+    } else {
+        CommonMessages.showError("couldNotDoAction", true);
```

Comment:
I am unsure, is `couldNotDoAction` really already defined?

Reply:
See above on lines 117-119. This is just switching the twisted if/else statements per the [no-negated-condition](https://eslint.org/docs/latest/rules/no-negated-condition) ESLint rule. The Unicorn ESLint plugin [includes an autofix](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-negated-condition.md) for this.